verilog/analysis/verilog_linter_configuration.cc

```diff
@@ -283,7 +298,12 @@ static absl::StatusOr<std::vector<std::unique_ptr<T>>> CreateRules(
     if (!setting.configuration.empty()) {
       if (absl::Status status = rule_ptr->Configure(setting.configuration);
           !status.ok()) {
-        return status;
+        std::string error_msg;
+
+        error_msg.append(std::string(rule_pair.first));
```

Comment:
`absl::StrCat()` is a good way to join strings in a readable way.
(also, this change only mildly looks like it should be part of this PR as it is not part of the rules_verible_lint printing.

Reply:
I hit this error message when I was miss configuring the rule above. The error message generated didn't link the fault back to a rule so I've prepended the rule name to help the user narrow down the issue better. The new error message should look like this:

`E0327 20:11:47.143266    2977 verilog_linter.cc:146] Fatal error: explicit-parameter-storage-type exempt_type: Value can only be one of ['', 'string']; got 'invalid_value'
`

For future reference, would you prefer this to come across as a seperate PR?

Reply:
Since it helped you discovering inadequate error reporting while working on this PR, this is good to keep in this PR. Just use `absl::StrCat()` to assemble the string.